site/en/blog/early-hints/index.md

```diff
@@ -150,7 +150,11 @@ Here is a quick summary of the level of support for Early Hints among popular OS
 If you are using one of the following CDNs or platforms, you may not need to manually implement Early Hints. Refer to your solution provider's online documentation to find out if it supports Early Hints, or refer to the non-exhaustive list here:
 
 - [Early Hints at Cloudflare](https://developers.cloudflare.com/cache/about/early-hints/)
-- [Early Hints at Fastly](https://www.google.com/url?q=https://www.fastly.com/blog/beyond-server-push-experimenting-with-the-103-early-hints-status-code%23:~:text%3Dabout%2520this%2520feature.-,Sending%2520103%2520Early%2520Hints,in%2520VCL%252C%2520like%2520this%253A&sa=D&source=docs&ust=1655290700718877&usg=AOvVaw07p23rkLcV4vplasZJBjEx).
+- [Early Hints at Fastly](https://www.fastly.com/blog/beyond-server-push-experimenting-with-the-103-early-hints-status-code#:~:text=about%20this%20feature.-,Sending%20103%20Early%20Hints,in%20VCL%2C%20like%20this%3A).
+
+## Avoiding issues for clients that do not support Early Hints
+
+Informational HTTP responses in the 100 range are part of the HTTP standard, but some older clients or bots may struggle with these as, prior to the launch of 103 Early Hints they were rarely used for general web browsing. Only emitting 103 Early Hints for clients that send a `sec-fetch-mode=navigate` HTTP header in the request should ensure these are only sent for clients that understand to wait for the subsequent response.
```

Comment:
```suggestion
Informational HTTP responses in the 100 range are part of the HTTP standard, but some older clients or bots may struggle with these because, prior to the launch of 103 Early Hints, they were rarely used for general web browsing. Only emitting 103 Early Hints in response to clients that send a `sec-fetch-mode=navigate` HTTP request header should ensure such hints are only sent for clients that understand to wait for the subsequent response.
```